spaceship/lib/spaceship/connect_api/users/users.rb

```diff
@@ -14,14 +14,50 @@ def users_request_client
         end
 
         #
-        # users
+        # users and invitations
         #
 
+        # Get all accepted users
         def get_users(filter: {}, includes: nil, limit: nil, sort: nil)
           params = users_request_client.build_params(filter: filter, includes: includes, limit: limit, sort: sort)
           users_request_client.get("users", params)
         end
 
+        # Get all invited users (not yet accepted)
+        def get_invited_users(filter: {}, includes: nil, limit: nil, sort: nil)
+          params = users_request_client.build_params(filter: filter, includes: includes, limit: limit, sort: sort)
+          users_request_client.get("userInvitations", params)
+        end
+
+        # Invite new users to App Store Connect
+        def invite_user(email: nil, first_name: nil, last_name: nil, roles: [], provisioning_allowed: nil, all_apps_visible: nil)
```

Comment:
```suggestion
        def post_user_invitation(attributes: [])
```

Most of the time I've been following this convention of naming and attribute passing and then make the helper method on the model.

In this case, I think we also need to add a `Spaceship::ConectAPI::UserInvitation` model for this to parse into (I don't think we have that yet). 

I'd be happy to help add to his PR or I think @bolom had one recently that did something similar 😊